addons/web/static/src/js/fields/res_partner_many2one.js

```diff
@@ -0,0 +1,15 @@
+odoo.define('web.res_partner_many2one', function (require) {
+    "use strict";
+
+    var fieldRegistry = require('web.field_registry');
+    var FieldMany2One = require('web.relational_fields').FieldMany2One;
+
+    var PartnerFieldMany2One = FieldMany2One.extend({});
```

Comment:
Two questions:
 - for now, there is only one override of this. Do you really need this generic mechanism? Can't you simply create that widget in partner_autocomplete?
 - if the answer of previous question is yes, can't you make FieldMany2one allowing extra autocompletes directly in this widget? Is there a need to make FieldMany2one that generic?